src/AutoMapper.Net4/AutoMapper.Net4.csproj

```diff
@@ -49,13 +49,487 @@
     <Reference Include="System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="..\AutoMapper\QueryableExtensions\Impl\ISourceInjectedQueryable.cs">
+      <Link>QueryableExtensions\Impl\ISourceInjectedQueryable.cs</Link>
+    </Compile>
     <Compile Include="..\CommonAssemblyInfo.cs">
       <Link>CommonAssemblyInfo.cs</Link>
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\AutoMapper\**\*.cs">
-      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    <Compile Include="..\AutoMapper\AliasedMember.cs">

```

Comment:
Oh man... can't remember having linked anything of that manually to the csproj...